src/connect4/ai.rs

```diff
@@ -2,196 +2,252 @@
 // [This program is licensed under the "MIT License"]
 // Please see the file LICENSE in the source
 // distribution of this software for license terms.
-/*
-use connect4::core;
+
+use connect4::core::GridPosition;
+use connect4::core::Board;
+use connect4::core::GameState;
+use connect4::core::BOARD_SIZE;
+use connect4::core::MyColor;
```

Comment:
I think can be simplified to `use connect4::core::{GridPosition, Board, GameState, Board_Size, MyColor};`